README.md

```diff
@@ -1,5 +1,30 @@
-# project-proposals-f2020
-Project Proposals for Fall 2020
+# WEAT
```

Comment:
dont modify readme, modify <uni>.md

Reply:
I thought that we'd need a README to display on github. Can I just delete this whole file then?